src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleOptions2.cs

```diff
@@ -359,8 +359,8 @@ private static string GetAccessibilityModifiersRequiredEditorConfigString(CodeSt
                 KeyValuePairUtil.Create("when_strongly_typed", ForEachExplicitCastInSourcePreference.WhenStronglyTyped),
             });
 
-        internal static readonly PerLanguageValuedOption2<CodeStyleOption2<ForEachExplicitCastInSourcePreference>> ForEachExplicitCastInSource =
-            CreateOption(
+        internal static readonly SingleValuedOption2<CodeStyleOption2<ForEachExplicitCastInSourcePreference>> ForEachExplicitCastInSource =
+            CreateCommonOption(
```

Comment:
I added an assert for per-language code-style that they should have a roaming location with `%LANGUAGE%`, which fails for this option. So I changed it to single-valued. It's already not really per-language. All languages will have the same value.

I also changed `OperatorPlacementWhenWrapping` while looking in this file for the same reason, but `OperatorPlacementWhenWrapping` doesn't hit the assertion as it's not used by analyzers.